layer/clients/data_catalog.py

```diff
@@ -206,7 +206,7 @@ def initiate_build(
             InitiateBuildRequest(
                 dataset_name=dataset.name,
                 format="python",
-                build_entity_type=PBDatasetBuild.BUILD_ENTITY_TYPE_DATASET,
+                build_asset_type=PBDatasetBuild.BUILD_asset_type_DATASET,
```

Comment:
are you sure you can do this? InitiateBuildRequest comes from proto files

Reply:
this is a bad change, fixing!